WalletWasabi.Fluent/ViewModels/Dialogs/ShuttingDownViewModel.cs

```diff
@@ -10,27 +12,35 @@ public partial class ShuttingDownViewModel : RoutableViewModel
 {
 	private readonly ApplicationViewModel _applicationViewModel;
 	private readonly bool _restart;
+	private readonly CoinJoinManager _coinJoinManager;
 
 	private ShuttingDownViewModel(ApplicationViewModel applicationViewModel, bool restart)
 	{
 		_applicationViewModel = applicationViewModel;
 		_restart = restart;
-		NextCommand = CancelCommand;
+		_coinJoinManager = Services.HostedServices.Get<CoinJoinManager>();
+		NextCommand = ReactiveCommand.CreateFromTask(
+			async () =>
+			{
+				await _coinJoinManager.RestartAbortedCoinjoinsAsync();
+				Navigate().Clear();
```

Comment:
This is only good until somebody not touches CancelCommand. 

Shouldn't be this more correct?

```suggestion
				CancelCommand.Execute(null);
```

Reply:
I really don't know. I didn't see the usage of `CancelCommand.Execute(null)` in the codebase before, so I went with this.

Maybe @soosr knows more about this.